test/MinimalDelegation.execute.t.sol

```diff
@@ -81,7 +83,7 @@ contract MinimalDelegationExecuteTest is TokenHandler, HookHandler, ExecuteFixtu
 
     function test_execute_auth_reverts() public {
         vm.expectRevert(IERC7821.Unauthorized.selector);
-        signerAccount.execute(BATCHED_CALL, abi.encode(CallUtils.initArray()));
+        signerAccount.execute(CallUtils.initArray(), true);
```

Comment:
oh man I don't like having the bool in the function sig

Reply:
ya same